Doc/whatsnew/3.10.rst

```diff
@@ -553,6 +553,11 @@ Build Changes
 * The :mod:`atexit` module must now always be built as a built-in module.
   (Contributed by Victor Stinner in :issue:`42639`.)
 
+* Added ``--enable-test-modules`` option to the ``configure`` script. If this
+  option is set to no, ``make install`` will not install test suites. Also setup.py
+  will not build test extension modules.
+  (Contributed by Peixing Xin in :issue:`27640`.)
```

Comment:
Xavier de Gaye wrote the original patch at 2016-07-28:  disable-test-suite.patch attached to https://bugs.python.org/issue27640

Thomas Petazzoni is maintaining a patch in Buildroot for 3 years: https://git.buildroot.org/buildroot/tree/package/python3/0013-Add-an-option-to-disable-installation-of-test-module.patch

It would be kind to also credit them:

```suggestion
  (Contributed by Xavier de Gaye, Thomas Petazzoni and Peixing Xin in :issue:`27640`.)
```

```
Co-Authored-By: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
Co-Authored-By: Xavier de Gaye <xdegaye@gmail.com>
```